research/object_detection/meta_architectures/faster_rcnn_meta_arch.py

```diff
@@ -1973,9 +1973,16 @@ def _compute_second_stage_input_feature_maps(self, features_to_crop,
     Returns:
       A float32 tensor with shape [K, new_height, new_width, depth].
     """
+    features_to_crop = [features_to_crop]
+    num_levels = len(features_to_crop)
+    box_levels = None
+    if num_levels != 1:
+      # If there are mutiple levels to select, get the box levels
+      box_levels = ops.fpn_feature_levels(num_levels, num_levels - 1,
+                                          1.0/224, proposal_boxes_normalized)
     cropped_regions = self._flatten_first_two_dimensions(
         self._crop_and_resize_fn(
```

Comment:
Please update other references of _crop_and_resize_fn in faster_rcnn_meta_arch.py

Reply:
Hi, I updated _crop_and_resize_fn in both faster_rcnn_meta_arch.py and context_rcnn_meta_arch.py. Currently, all the tests except the ones related to pycocotools/cocoeval.py and used TPU are passed.